### PR TITLE
working-with-npm-milestone

### PR DESCRIPTION
**Pull request by vikispeaks**

### CODE REVIEW

File: working-with-npm/milestone/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Text
+
+**Important Notes:**
+>
+> 1. You must format the code using `prettier` before submitting it.
+> 2. You must read the problem description carefully before making your submission. Do not miss the details - the program should perfectly match the specifications.
+> 3. Take your time, sweat the details, and make every submission count!
+
+## Problem Description
+To complete this milestone, you have to make two updates to the **Node.js** program you created at this level, as per the details given.
```
Comment (avishekjana) [inline working-with-npm/milestone/README.md:10]: To complete this milestone, you have to make two updates to the **Node.js** program you've created in this level, as per the details given.

File: working-with-npm/milestone/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+1. Create a new HTML page to showcase your previous project HTML. (You can make use of the `survey.html` you created in WD101 and name the file the same.)
```
Comment (avishekjana) [inline working-with-npm/milestone/README.md:12]: 1. Create a new HTML page to showcase your previous project in HTML. (You can make use of the `survey.html` you created in WD101 and name the file the same.)

File: working-with-npm/milestone/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+2. Add a Link to the Projects list page to point to the Survey page.
```
Comment (avishekjana) [inline working-with-npm/milestone/README.md:13]: 2. Add a **Link** to the _Projects list_ page to point to the _Survey page_.

File: working-with-npm/milestone/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+3. Create a new route to the Survey page and make sure the navigation from the Projects page works fine.
```
Comment (avishekjana) [inline working-with-npm/milestone/README.md:14]: Can the word "route" be replaced with something else?

**Comment by reenas** [comment]
@vikispeaks Can you address resolve the comments, so this PR is complete?

File: working-with-npm/milestone/README.md (hunk 1 above)
Comment (vikispeaks) [reply working-with-npm/milestone/README.md:10]: Done.

File: working-with-npm/milestone/README.md (hunk 2 above)
Comment (vikispeaks) [reply working-with-npm/milestone/README.md:12]: Keeping it same to match the reference.

File: working-with-npm/milestone/README.md (hunk 4 above)
Comment (vikispeaks) [reply working-with-npm/milestone/README.md:14]: We already spoke about what a route means in earlier lessons. I think we can keep it the same.

**Comment by vikispeaks** [comment]
@reenas Fixed and Merged.

**Pull request merged**